packages/web-runtime/src/store/app.ts

```diff
@@ -7,23 +7,24 @@ const state = {
 
 const actions = {
   showErrorMessage({ commit }, message) {
-    const getXRequestID = (error: AxiosError): string => {
-      if (!error) {
-        return
-      }
+    const getXRequestID = (error: AxiosError): string | null => {
       if (error.response?.headers?.['x-request-id']) {
         return error.response.headers['x-request-id']
       }
+      return null
     }
+
     message.status = message.status || 'danger'
     message.timeout = message.timeout || 0
+    message.errors = message.error ? [message.error] : message.errors || []
```

Comment:
@dschmidt you can pass errors or error to showErrorMessage. 
Should we purge error and rely on the developer passes errors:[e] to the function